app/policies/application_policy.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def index?
+    false
+  end
+
+  def show?
+    false
+  end
+
+  def create?
+    false
+  end
+
+  def new?
+    create?
+  end
```

Comment:
As we are doing API only application we will not have in our controllers actions for `new` and `edit` - only for `update` and `create`, so you can drop both from your policies